ezgui/src/drawing.rs

```diff
@@ -251,4 +254,20 @@ impl Prerender {
     pub(crate) fn request_redraw(&self) {
         self.inner.request_redraw()
     }
+
+    pub fn get_scale_factor(&self) -> f64 {
+        *self.scale_factor.borrow()
```

Comment:
This was moved from Assets. 

I think it used to live on Assets because it was needed for svg loading and layout tweaks (via EventCtx). Now that we don't use scale_factor in those places, I've moved it down.